### ISSUE TITLE
D_tc_loss line 119 in solver.py

### ISSUE THREAD

**Issue by goodman1204** [1Konny/FactorVAE#14]
I am not sure about the following code, can someone help to explain it?  
D_tc_loss = 0.5*(F.cross_entropy(D_z, zeros) + F.cross_entropy(D_z_pperm, ones))

**Comment by charan223** [comment]
same doubt here

**Comment by charan223** [comment]
They are just increasing the values of label 0 for D_z, and label 1 for D_z_pperm
